docs/v1/P4Runtime-Spec.adoc

```diff
@@ -2211,73 +2034,88 @@ set, a single entry will be read.
 Let's look at the counter example in more details. Based on this specification
 document, the C++ server code which processes `CounterEntry` messages may look
 like this:
-~ Begin CPP
+
+[source,c++]
+----
 auto *counter_entry = ...
 if (counter_entry->has_index()) {
   auto index = counter_entry->index().index();
   read_one_entry(counter_entry->id(), index);
 } else {
   read_all_entries(counter_entry->id());
 }
-~ End CPP
+----
 
-1. Reading a single counter entry at index 0 in the counter array with id
+. Reading a single counter entry at index 0 in the counter array with id
```

Comment:
In the PDF, there is a "1." appearing before the text and it appears elevated. But the source looks like this was corrected?

Reply:
A dot at the beginning of a line is one way in AsciiDoc to distinguish a numbered list item.  See here: https://docs.asciidoctor.org/asciidoc/latest/lists/ordered/